src/message_private.h

```diff
@@ -113,6 +113,7 @@ typedef struct {
     bool                 receive_complete;               // true if the message has been completely received, false otherwise
     sys_atomic_t         fanout;                         // The number of receivers for this message. This number does not include in-process subscribers.
     bool                 q2_input_holdoff;               // hold off calling pn_link_recv
+    bool                 aborted;                        // receive completed with abort flag set
```

Comment:
For performance with smaller cache use it would make sense to group the bools together and not split them with elements that have alignment requirements.